app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -1110,6 +1111,7 @@ val MAP_SORT_TYPE_LONG_FORM = mapOf(
     SortType.Hot to R.string.sorttype_hot,
     SortType.New to R.string.sorttype_new,
     SortType.Old to R.string.sorttype_old,
+    SortType.Controversial to R.string.sorttype_controversial,
```

Comment:
Could you move both `SHORT_FORM` and `LONG_FORM` to be part of the sortType enum? and then remove the getLocalized helper functions?